src/javascript/binary/websocket_pages/user/ico_subscribe.js

```diff
@@ -109,6 +110,31 @@ const ICOSubscribe = (() => {
         $total.html(formatMoney(currency, total));
     };
 
+    const ICOcountDown = () => {
+        const timer = window.setInterval(() => {
+            const start_time = 1510704000; 
+            const current_time = Math.round((new Date()).getTime()/1000); 
+            const time_left = start_time - current_time; 
+            if(time_left >= 0) {
+                $('.timer').removeClass('invisible'); // Make the timer visible.
+                const s = (`0${  time_left % 60}`).slice(-2); 
+                const m = (`0${  Math.floor(time_left/ 60) % 60}`).slice(-2); 
+                const h = (`0${  Math.floor(time_left / 3600) % 24}`).slice(-2); 
+                const d = (`0${  Math.floor(time_left / (3600 * 24))}`).slice(-2); 
+                $('.timer .time .days').text(d);
```

Comment:
we can just assign `const $timer =  $('.timer'); const $time = $timer.find('.time'); const $days = $time.find('.days'); const $hours = $time.find('.hours');` and so on outside of interval function and call the variables here.